Python/omevv/v1/scripts/CreateRepoProfile.py

```diff
@@ -0,0 +1,122 @@
+);
+parser.add_argument('-ip', help='OMEVV IP address', required=True);
+parser.add_argument('-ome_user', help='OME username', required=True);
+parser.add_argument('-ome_pswd', help='OME password', required=True);
+parser.add_argument('-omevv_user', help='OMEVV username', required=True);
+parser.add_argument('-omevv_pswd', help='OMEVV password', required=True);
+parser.add_argument('-profilename', help='repository profile name', required=True);
+parser.add_argument('-description', help='repository profile description', required=False);
+parser.add_argument('-profileType', help='repository profile type', required=True);
+parser.add_argument('-sharepath', help='sharepath', required=True);
+parser.add_argument('-spcred', help='sharepath credential', required=True);
+parser.add_argument('-protocol',help='protocol type',choices=['CIFS','NFS','HTTP','HTTPS','UNAVAILABLE'],required = True);
+parser.add_argument('-certificateCheck',help ='certificate check required?',required = False);
+
+args = vars(parser.parse_args())
+ome_ip = args['ip'];
+ome_user_name = args['ome_user'];
+ome_cred = args['ome_pswd'];
+omevv_user_name = args['omevv_user'];
+omevv_cred = args['omevv_pswd'];
+retry = 3;
+payload ={};
+
+def create_payload():
+    if args["profilename"]:
+        payload["profileName"] = args["profilename"];
+    if args["description"] :
+       payload["description"] = args["description"];
+    if args["profileType"]:
+        payload["profileType"] = args["profileType"];
+    if args["sharepath"]:
+        payload["sharePath"] = args["sharepath"];
+    if args["protocol"]:
+        payload["protocolType"] = args["protocol"];
+
+    if args['protocol'] == "CIFS" and args['spcred'] == "":
+        raise Exception('credential missing for sharedpath for CISF protocol')
+    if args['spcred']:
+        creds = args['spcred'].split("|");
+        userName = creds[0];
+        pswd = creds[1];
+        spcred = {"username": userName.strip(),"password": pswd.strip()};
+        payload['shareCredential'] = spcred
+
+    if args["certificateCheck"]:
+        payload["checkCertificate"] = args["certificateCheck"];
+
+
+
+
+
+
+
+
+def get_uuid():
+    uuid = "";
+    url = "https://%s/omevv/GatewayService/v1/Consoles"%(ome_ip);
+    encoded_cred = encode_cred(ome_user_name,ome_cred);
+    cred_str = 'Basic '+encoded_cred;
+    headers = {'Authorization': cred_str};
+    response = requests.get(url,headers=headers,verify=False);
+    status_code = response.status_code
+    if status_code != 200:
+       print("Error:Failed to retrieve the UUID ",response.content);
+    else:
+        data = response.json();
+        uuid = data[0]['uuid']
+    return uuid;
+
+def encode_cred(username,pwd):
+    cred_str = username+":"+pwd;
+    cred_str_bytes = cred_str.encode("ascii");
+    base64_bytes = base64.b64encode(cred_str_bytes);
+    base64_string = base64_bytes.decode("ascii");
+    return base64_string;
+
+def create_repo_profl():
```

Comment:
Be great if this method can return the repo id for successful execution and throws an exception in case of an error that can be handled by scripts which import and use this method.